packages/ckeditor5-table/src/tableutils.js

```diff
@@ -289,17 +289,21 @@ export default class TableUtils extends Plugin {
 		const last = first + rowsToRemove - 1;
 		const batch = options.batch || 'default';
 
-		// Removing rows from table requires most calculations to be done prior to changing table structure.
+		model.enqueueChange( batch, writer => {
+			// Removing rows from table requires most calculations to be done prior to changing table structure.
+			// Preparations must be done in enqueueChange callback to use the current table structure.
```

Comment:
```suggestion
			// Removing rows from the table require that most calculations to be done prior to changing table structure.
			// Preparations must be done in the same enqueueChange callback to use the current table structure.
```

Minor wording change.